pkg/v1/providers/infrastructure-aws/v1.2.0/ytt/overlay.yaml

```
@@ -188,6 +198,8 @@ spec:
   replicas: #@ data.values.WORKER_MACHINE_COUNT_0
   selector:
     matchLabels: null
+  strategy:
+    type: #@ verify_and_configure_strategy(data.values.WORKER_ROLLOUT_STRATEGY)
```

```suggestion
    type: #@ verify_and_configure_machine_deployment_rollout_strategy(data.values.WORKER_ROLLOUT_STRATEGY)
```

`verify_and_configure_strategy` doesn't have the context of where it's happening.